### DISCUSSION TITLE
How to git download just one release directory?

### DISCUSSION THREAD

**Discussion by gitclem** [schemaorg/schemaorg#2872] [Q&A]
I found this, but there were many different recommendations, so I wasn't sure which one to try:

https://stackoverflow.com/questions/600079/how-do-i-clone-a-subdirectory-only-of-a-git-repository/52269934

Is there a way to just download part of the schemaorg git repo?

E.g. If I want [release 12](https://github.com/schemaorg/schemaorg/tree/main/data/releases/12.0), is there a recommended way to download just release 12?  (I don't think I care about all the git stuff that would come with it, so just the files.)

In the meantime, I'll experiment with the various suggestions in Stackoverflow.

Also, a newbie, like myself, it would be nice to also have an example from each file from the release, so that I can more easily determine which files I actually want to download.

TIA.

**Comment by RichardWallis** [comment]
GitHub is not very good at such things - it assumes that the contents of the repo are an interconnected whole.

I am assuming that you are trying to get a download of the term definitions for a release, not the internal files that go to building a release.  

If that is the case, a good place to start would be the Schema.org [developers page](https://schema.org/docs/developers.html#defs) where you can download a single* file (in a format of your choice)  that contains those definitions.

You can select your preference for terms defined using http or https URIs, then which serialisation you require (XML, JSON-LD, etc.)

Beyond that there is a choice of the 'current' terms or 'all' terms.  If you select 'all' the file will include terms that have been previously removed from the vocabulary (known as being placed into the 'attic' area).

This facility operates for the latest release of the vocabulary (currently version 12.0)  Once you have identified the file/format that is useful for you, that file for non-latest releases can be manually downloaded from the relevant release directory in Github.

Hope this helps.

*_CSV format definitions are included in two files._

**Reply by thadguidry** [reply]
@gitclem the "release directory" that @RichardWallis means is always under `/schemaorg/data/releases` in the repo, Ex. https://github.com/schemaorg/schemaorg/tree/main/data/releases/13.0 .  There is also a separate special page that GitHub has to hold Releases (there's also one for Tags) Ex. https://github.com/schemaorg/schemaorg/releases
You can download the Source code and extract it fully from the Asset `.zip` files as well, if need be.
